rpcs3/Emu/SysCalls/Modules/sys_mempool.cpp

```diff
@@ -1,40 +1,129 @@
 #include "stdafx.h"
 #include "Emu/Memory/Memory.h"
 #include "Emu/System.h"
+#include "Emu/IdManager.h"
 #include "Emu/SysCalls/Modules.h"
 
 #include "sysPrxForUser.h"
 
 extern Module sysPrxForUser;
 
+using sys_mempool_t = u32;
+
+struct memory_pool_t
```

Comment:
@KaOs1 It is overall still prefered to put all the typdefs/structs/etc into headers to keep the .cpp from being "bloated" (especially, when there are a lot of typdefs/structs/etc). Plus it would be nice to keep it consistent, with rest of the emulator.